src/java/us/kbase/templates/perl_server.vm.properties

```diff
@@ -261,7 +261,14 @@ sub call_method {
 	    $self->exception('PerlError', "Authentication required for ${last_module.module_name} but no authentication header was passed");
 	}
 
-	my $auth_token = Bio::KBase::AuthToken->new(token => $token, ignore_authrc => 1);
+	my $auth_token;
+        if ($self->config->{'auth-service-url'})
+        {
+            $auth_token = Bio::KBase::AuthToken->new(token => $token, ignore_authrc => 1, auth_svc=>$self->config->{'auth-service-url'});
+        } else {
+            $auth_token = Bio::KBase::AuthToken->new(token => $token, ignore_authrc => 1);
+        }
+
```

Comment:
here's a relevant comment from the last PR with these changes
> I want to run three different permutations of test.

>    Current codebase
>    These changes plus stock auth (should work, I think)
>    These changes plus Perl auth client changes in auth/dev-perl_auth branch

> So far test 1. runs for a long time, so I may need to kill it and start again tomorrow. 3. is having problems, and I need to track down exactly what they are. It's currently failing in testJavaAsyncService in AsyncDockerTest, which is odd because I didn't touch that code. It may be a different issue.

So changes may be necessary to get tests to pass.